tests/integration/Erfurt/StoreIntegrationTest.php

```diff
@@ -360,7 +360,63 @@ public function testImportDBPediaResourceMachineLearningAndQueryWithTitleHelperQ
         $this->assertInternalType('array', $sparqlResult);
         $this->assertNotEmpty($sparqlResult);
     }
-}
 
+    public function testDeleteWithLongLiteralsIssue303OntoWiki()
+    {
+        $this->markTestNeedsDatabase();
+        $this->authenticateDbUser();
+
+        $modelUri = 'http://example.org/updateTest/';
+        $store = Erfurt_App::getInstance()->getStore();
+        try {
+            $store->deleteModel($modelUri);
+        } catch (Exception $e) {/*ignore deletion failure*/}
+        $model = $store->getNewModel($modelUri);
+        $literal= <<<'EOT'
+"""Gerade eben habe ich einen kleinen PHP Skript geschrieben, der sich selbst als Bild einbindet. Das klingt etwas seltsam aber veranschaulicht das Prinzip der <a href=\"http://de.wikipedia.org/wiki/Content_Negotiation\"><em>Content Negotiation</em></a> sehr gut.
```

Comment:
Is it possible to switch to example texts in English instead of German for future work?